.github/workflows/oci.yml

```diff
@@ -0,0 +1,228 @@
+name: oci-release
```

Comment:
Let's make sure to place all reusable workflows to a consistent location (`.github/actions`)